src/Libraries/RevitNodes/Elements/Views/FloorPlanView.cs

```diff
@@ -60,6 +52,18 @@ private void InitFloorPlanView(Autodesk.Revit.DB.Level level)
 
         #endregion
 
+        #region Public constructors
+
+        /// <summary>
+        /// Create a Revit Floor Plan from Autodesk View Plan
+        /// </summary>
+        public  FloorPlanView(Autodesk.Revit.DB.ViewPlan view)
```

Comment:
this likely should not be public - or it should at least be hidden from the node library.

Reply:
When I did the merge, I didn't pay attention to ZiyunShang commit ( revert public constructor to private (#2523)) where the fix for using default view in tests without using the private constructor of FloorPlanView was implemented.
Changes reverted back to ZiyunShang  commit.
Thanks and sorry.